### PR TITLE
Popup: update scrolling demo

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

**Review by Zedwag — DISMISSED** [review]

**Review by marker-dao — DISMISSED** [review]

File: JSDemos/Demos/Popup/Scrolling/jQuery/styles.css [hunk 1]
```diff
@@ -1,13 +1,44 @@
...
+  flex-direction: column;
+  justify-content: center;
+  gap: 15px;
+}
+
+.dx-popup-content {
+  font-size: 12px;
+}
+
+.dx-popup-title .dx-toolbar-before {
```
Comment (ksercs) [inline JSDemos/Demos/Popup/Scrolling/jQuery/styles.css]: Why do we need this style?
It looks like we're working around some internal restriction

File: JSDemos/Demos/Popup/Scrolling/jQuery/index.html [hunk 2]
```diff
@@ -13,6 +13,44 @@
     <script src="index.js"></script>
   </head>
   <body class="dx-viewport">
-    <div id="popup"></div>
+    <div class="demo-container">
+      <div class="button-container">
+        <div id="show-scrollable-popup"></div>
```
Comment (ksercs) [inline JSDemos/Demos/Popup/Scrolling/jQuery/index.html]: We discussed the naming with Vlada. 
Let's use the following naming: just a "popup" and a "popupWithScrollView". 
I also ask u to make the naming consistent across the frameworks.

Points:
- "scrollViewPopup" is not very good from a semantic point view
- "scrollable" and so on is a long, similar to "scrollable" component from docs
- we wanna emphasize the default behavior

```suggestion
        <div id="show-popup-button"></div>
```

Let's also ask Vlada for a final review after all changes

File: JSDemos/Demos/Popup/Scrolling/jQuery/index.js [hunk 3]
```diff
@@ -1,22 +1,80 @@
...
-      scrollView.dxScrollView({
-        width: '100%',
-        height: '100%',
-      });
+  $('#show-scrollable-popup').dxButton({
+    text: 'Show Popup',
+    width: 300,
+    type: 'default',
+    onClick() {
+      popupWithScrollableContainer.show();
```
Comment (ksercs) [inline JSDemos/Demos/Popup/Scrolling/jQuery/index.js]: Naming also affects the code, not only a markup.
```suggestion
      popup.show();
```

File: JSDemos/Demos/Popup/Scrolling/jQuery/index.js [hunk 4]
```diff
@@ -1,22 +1,80 @@
...
+            text: 'Book',
+            type: 'default',
+            onClick() {
+              popupWithScrollView.hide();
+            },
+          },
+        },
+      ],
+      contentTemplate() {
+        const scrollView = $('<div />');
```
Comment (ksercs) [inline JSDemos/Demos/Popup/Scrolling/jQuery/index.js]:
```suggestion
        const $scrollView = $('<div/>');
```

File: testing/widgets/common/DialogsAndNotificationsOverview.test.js [hunk 5]
```diff
@@ -16,7 +16,7 @@ runManualTest('Common', 'DialogsAndNotificationsOverview', ['jQuery', 'React', '
   test('DialogsAndNotificationsOverview', async (t) => {
     const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
 
-    await t.click($(`.${IMAGES_CONTAINER_CLASS} .${ITEM_CONTENT_CLASS} img`));
+    await t.click($(`.${IMAGES_CONTAINER_CLASS} .${ITEM_CONTENT_CLASS}`));
```
Comment (ksercs) [inline testing/widgets/common/DialogsAndNotificationsOverview.test.js]: Why do we need change this?

File: JSDemos/Demos/Popup/Scrolling/Vue/App.vue [hunk 6]
```diff
@@ -1,65 +1,191 @@
...
-          platforms, assign <b>true</b> to the <b>useNative</b> option. If you assign
-          <b>false</b>, scrolling is simulated on all platforms.
+  <div>
+    <div class="demo-container">
+      <div class="button-container">
+        <DxButton
+          :width="300"
+          text="Show Popup"
+          type="default"
+          @click="onShowScrollablePopup($event)"
```
Comment (ksercs) [inline JSDemos/Demos/Popup/Scrolling/Vue/App.vue]:
```suggestion
          @click="showPopup($event)"
```

File: MVCDemos/Views/Popup/Scrolling.cshtml [hunk 7]
```diff
@@ -1,22 +1,140 @@
...
+                    <div>Iron/ironing</div>
+                </div>
             </div>
         </text>)
     )
 }
+
+<script>
+    function showScrollablePopup() {
+        $("#scrollable-popup").dxPopup("instance").show();
```
Comment (alexanderPolosatov) [inline MVCDemos/Views/Popup/Scrolling.cshtml]: can be replaced to
$("#scrollable-popup").dxPopup("show");

Comment (alexanderPolosatov) [reply MVCDemos/Views/Popup/Scrolling.cshtml]: and in other places too

File: testing/widgets/common/DialogsAndNotificationsOverview.test.js (hunk 5 above)
Comment (EugeniyKiyashko) [reply testing/widgets/common/DialogsAndNotificationsOverview.test.js]: Yes, it's unsufficient selector for testcafe. This change allow prevent warning appearence and blinking of the test.

**Review by alexanderPolosatov — DISMISSED** [review]

File: JSDemos/Demos/Popup/Scrolling/jQuery/index.html (hunk 2 above)
Comment (EugeniyKiyashko) [reply JSDemos/Demos/Popup/Scrolling/jQuery/index.html]: Renamed in [fix notes after review](https://github.com/DevExpress/devextreme-demos/pull/2486/commits/0f1b08d8909b276906184e23e235d9bfc1f08533). @vladaskorohodova  check it please again

File: JSDemos/Demos/Popup/Scrolling/jQuery/styles.css (hunk 1 above)
Comment (EugeniyKiyashko) [reply JSDemos/Demos/Popup/Scrolling/jQuery/styles.css]: There was some reason to add this css rule. It's not require anymore and I removed it.

File: JSDemos/Demos/Popup/Scrolling/jQuery/index.html (hunk 2 above)
Comment (EugeniyKiyashko) [reply JSDemos/Demos/Popup/Scrolling/jQuery/index.html]: Fixed

File: JSDemos/Demos/Popup/Scrolling/jQuery/index.js (hunk 4 above)
Comment (EugeniyKiyashko) [reply JSDemos/Demos/Popup/Scrolling/jQuery/index.js]: Fixed

File: JSDemos/Demos/Popup/Scrolling/Vue/App.vue (hunk 6 above)
Comment (EugeniyKiyashko) [reply JSDemos/Demos/Popup/Scrolling/Vue/App.vue]: Renamed

File: MVCDemos/Views/Popup/Scrolling.cshtml (hunk 7 above)
Comment (EugeniyKiyashko) [reply MVCDemos/Views/Popup/Scrolling.cshtml]: Ok, great advice to do this construction shorter

File: JSDemos/Demos/Popup/Scrolling/jQuery/index.js (hunk 3 above)
Comment (EugeniyKiyashko) [reply JSDemos/Demos/Popup/Scrolling/jQuery/index.js]: Fixed

**Review by Zedwag — APPROVED** [review]

**Pull request merged**